src/native.ts

```diff
@@ -0,0 +1,171 @@
+    target: 'mipmap-mdpi/ic_launcher_round.png',
+  },
+  {
+    source: 'drawable-mdpi-icon.png',
+    target: 'mipmap-mdpi/ic_launcher_foreground.png',
+  },
+  { source: 'drawable-hdpi-icon.png', target: 'mipmap-hdpi/ic_launcher.png' },
+  {
+    source: 'drawable-hdpi-icon.png',
+    target: 'mipmap-hdpi/ic_launcher_round.png',
+  },
+  {
+    source: 'drawable-hdpi-icon.png',
+    target: 'mipmap-hdpi/ic_launcher_foreground.png',
+  },
+  { source: 'drawable-xhdpi-icon.png', target: 'mipmap-xhdpi/ic_launcher.png' },
+  {
+    source: 'drawable-xhdpi-icon.png',
+    target: 'mipmap-xhdpi/ic_launcher_round.png',
+  },
+  {
+    source: 'drawable-xhdpi-icon.png',
+    target: 'mipmap-xhdpi/ic_launcher_foreground.png',
+  },
+  {
+    source: 'drawable-xxhdpi-icon.png',
+    target: 'mipmap-xxhdpi/ic_launcher.png',
+  },
+  {
+    source: 'drawable-xxhdpi-icon.png',
+    target: 'mipmap-xxhdpi/ic_launcher_round.png',
+  },
+  {
+    source: 'drawable-xxhdpi-icon.png',
+    target: 'mipmap-xxhdpi/ic_launcher_foreground.png',
+  },
+  {
+    source: 'drawable-xxxhdpi-icon.png',
+    target: 'mipmap-xxxhdpi/ic_launcher.png',
+  },
+  {
+    source: 'drawable-xxxhdpi-icon.png',
+    target: 'mipmap-xxxhdpi/ic_launcher_round.png',
+  },
+  {
+    source: 'drawable-xxxhdpi-icon.png',
+    target: 'mipmap-xxxhdpi/ic_launcher_foreground.png',
+  },
+];
+const ANDROID_SPLASHES: readonly ProcessItem[] = [
+  { source: 'drawable-land-mdpi-screen.png', target: 'drawable/splash.png' },
+  {
+    source: 'drawable-land-mdpi-screen.png',
+    target: 'drawable-land-mdpi/splash.png',
+  },
+  {
+    source: 'drawable-land-hdpi-screen.png',
+    target: 'drawable-land-hdpi/splash.png',
+  },
+  {
+    source: 'drawable-land-xhdpi-screen.png',
+    target: 'drawable-land-xhdpi/splash.png',
+  },
+  {
+    source: 'drawable-land-xxhdpi-screen.png',
+    target: 'drawable-land-xxhdpi/splash.png',
+  },
+  {
+    source: 'drawable-land-xxxhdpi-screen.png',
+    target: 'drawable-land-xxxhdpi/splash.png',
+  },
+  {
+    source: 'drawable-port-mdpi-screen.png',
+    target: 'drawable-port-mdpi/splash.png',
+  },
+  {
+    source: 'drawable-port-hdpi-screen.png',
+    target: 'drawable-port-hdpi/splash.png',
+  },
+  {
+    source: 'drawable-port-xhdpi-screen.png',
+    target: 'drawable-port-xhdpi/splash.png',
+  },
+  {
+    source: 'drawable-port-xxhdpi-screen.png',
+    target: 'drawable-port-xxhdpi/splash.png',
+  },
+  {
+    source: 'drawable-port-xxxhdpi-screen.png',
+    target: 'drawable-port-xxxhdpi/splash.png',
+  },
+];
+
+async function copyImages(sourcePath: string, targetPath: string, images: readonly ProcessItem[], logstream: NodeJS.WritableStream) {
+  await Promise.all(images.map(async item => {
+    const source = path.join(sourcePath, item.source);
+    const target = path.join(targetPath, item.target);
+    await copy(source, target);
+    logstream.write(`Copied resource item from ${source} to ${target}\n`);
```

Comment:
This will be too verbose for regular logs. Please use the `debug` module (see other files for usage).

I think we should use `logstream` to print a message with aggregated results at the end, e.g. "Copied 20 resources to iOS"